tempodb/backend/s3/s3.go

```diff
@@ -107,13 +107,16 @@ func internalNew(cfg *Config, confirm bool) (backend.RawReader, backend.RawWrite
 func (rw *readerWriter) Write(ctx context.Context, name string, keypath backend.KeyPath, data io.Reader, size int64, _ bool) error {
 	objName := backend.ObjectFileName(keypath, name)
 
+	putObjectOptions := minio.PutObjectOptions{PartSize: rw.cfg.PartSize}
```

Comment:
super nit. it would look nicer to do:

```
putObjectOptions := minio.PutObjectOptions{
  PartSize: rw.cfg.PartSize,
  UserTags: rw.cfg.Tags,
}
```